.github/workflows/stale.yml

```diff
@@ -10,13 +10,15 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@v5
+      - uses: actions/stale@v7
         with:
           days-before-issue-stale: 90
           days-before-issue-close: -1
           stale-issue-label: "stale"
+          stale-pr-label: "stale"
+          stale-pr-message: "🚧🚨 This Pull Request is being marked as stale due to 30 days of inactivity. 🚧🚨"
           stale-issue-message: "🚧🚨 This issue is being marked as stale due to 90 days of inactivity. 🚧🚨"
-          days-before-pr-stale: -1
+          days-before-pr-stale: 30
           days-before-pr-close: -1
-          only-labels: 'needs triage'
+          only-labels: 'needs triage, waiting-review'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏so currently it will mark pull requests as "stale" only if they have "waiting-review" label. What if the label has been removed?

IMO we should create a new job for pull requests and then apply this flow without the `only-labels` filter (so it applies to all PRs no matter what labels they have).

WDYT?

Reply:
I can make the change. Sorry.

Reply:
This should be resolved.